src/renderer.tsx

```diff
@@ -481,18 +480,21 @@ function switchInstance(instance: any, type: string, newProps: any, fiber: Recon
 //    TimeoutHandle,
 //    NoTimeout
 //  >
-const Renderer = Reconciler({
+const Renderer = Reconciler<any, any, THREE.Object3D, any, any, any, any, any, any, any, any, any>({
```

Comment:
I had no time to fill this types in. If somebody have time, it would be great to improve, or just create a TODO for next issue.

Reply:
Probably best to wait til the reconciler is updated and we can fix these properly.

Reply:
in this case I think it is better to comment out `<>` completely, or comment out.

Reply:
@JakeSidSmith @dr sorry for off-topic, I'm looking for remote position, do you hire? Would love to bring my skill to the company which is working with react-three-fiber. I have skills with react, typescript, AWS cloud, Node.js, Security and Architecture.

I have build this: https://hexmate.netlify.app/ 
please open only on desktop, I'm in progress of building interface and did not adjust for mobile yet.
press start button on the left.

Feedback is welcome.

Reply:
My company doesn't work with r3, I'm just using it for some personal projects.

I have just opened a PR here: https://github.com/pmndrs/react-three-fiber/pull/840

Just making a few changes to it (including removing those generics).

P.s. you've tagged the wrong person? 😉

Reply:
Thank you for response, It is just blind question. Just tired from linkedin hr flow, and trying direct approach.

Reply:
I don't really have much to do with our hiring process, but we're currently hiring senior React devs.

Not sure if this will be permanently remote, but I can ask my boss on Monday.

https://www.dabapps.com/careers/senior-engineer-python-javascript-or-mobile/